test/test_fx.py

```diff
@@ -35,6 +35,7 @@
 from fx.test_subgraph_rewriter import TestSubgraphRewriter  # noqa: F401
 from fx.test_dce_pass import TestDCE  # noqa: F401
 from fx.test_fx_const_fold import TestConstFold  # noqa: F401
+from fx.test_gradual_type import AnnotationsTest  #  noqa: F401
```

Comment:
As discussed offline, because python below 3.7 does not have `__class_getitem__`, let's disable this feature for unsupported python versions. Let's gate this like so:

```
if sys.version_info >= (3, 7):
  from fx.test_gradual_type import AnnotationsTest  #  noqa: F401
```